pkg/certificate/identity.go

```diff
@@ -399,7 +399,7 @@ func PrepareRoleCsrOptions(idCfg *config.IdentityConfig, domain, service string)
 	var roleCsrOptions []util.CSROptions
 
 	if !idCfg.RoleCert.Use {
-		log.Debugf("Skipping to prepare csr for role certificates with target roles[%s], output directory[%s]", idCfg.RoleCert.TargetDomainRoles, idCfg.RoleCert.Dir)
+		log.Debugf("Skipping to prepare csr for role certificates with target roles[%s], filename format[%s]", idCfg.RoleCert.TargetDomainRoles, idCfg.RoleCert.Format)
```

Comment:
As long as we are aware that we should not have this line changed in this PR, we are good I think.

Reply:
(Or if you want you can make a separate PR, just for the logger)